components/messages/HowTo.module.css

```
@@ -0,0 +1,27 @@
+.howtocontainer {
```

Use camel case to make the class easier to read
```suggestion
.howToContainer {
```